src/System.Linq/src/System/Linq/Aggregate.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
A question, not a comment: Do we have a policy on whether unnecessary `using System` is to be removed? I've been leaving these any time I removed unnecessary usings.